host/3.0/buster/amd64/python/python39/python39.Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN EXTENSION_BUNDLE_VERSION=1.8.1 && \
     rm -f /$EXTENSION_BUNDLE_FILENAME_V3 &&\
     find /FuncExtensionBundles/ -type f -exec chmod 644 {} \;
 
-FROM python:3.9-slim-buster
+FROM mcr.microsoft.com/mirror/docker/library/python:3.9-buster
```

Comment:
Make -slim. Won't comment on other images. But please apply this comment to all nonsuffix images.